utils/index.js

```diff
@@ -288,9 +297,38 @@ const getPreviousRoute = (req, routes = defaultRoutes) => {
     return oneRouteBack
   }
 
+  if(req.query && req.query.ref) {
+    //I'm assuming here we want to let them go back to yes/no
+    if( req.path.includes('amount') ) {
+      return {path: `${prevRoute().path}?ref=checkAnswers`}
+    }
+    return routes.find(route => route.path === '/checkAnswers')
+  } 
+
   return prevRoute()
 }
 
+const returnToCheckAnswers = (req, res, claimYes = false, routes = defaultRoutes) => {
```

Comment:
Do we need this param: `routes = defaultRoutes`? Looks like nobody is using it.

Reply:
`const nextRoute = routes[currentRoute.index + 1]`

Reply:
we're using it for checking the nextRoute

Reply:
Yeah, but we don't need it to be a param though, right? On line 316, you could just say `defaultRoutes[currentRoute.index + 1]`